emperor/support_files/js/color-editor.js

```diff
@@ -0,0 +1,133 @@
+/**
+ *
+ * @author Yoshiki Vazquez-Baeza
```

Comment:
We removed these from the Python files but we didn't from the JavaScript
files, not sure what to do though. Others have suggestions?

On (Aug-01-15| 6:00), Antonio Gonzalez wrote:
> > @@ -0,0 +1,133 @@
> > +/**
> > + *
> > + * @author Yoshiki Vazquez-Baeza
> 
> Just out of curiosity, are we adding these headers to all scripts? Just wondering as we remove them from the python code ... 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/407/files#r36033367